packages/cli/src/commands/generate/sdl/sdl.js

```diff
@@ -225,6 +225,12 @@ export const handler = async ({ model, crud, force, tests, typescript }) => {
           return writeFilesTask(f, { overwriteExisting: force })
         },
       },
+      {
+        title: `Generating types ...`,
+        task: async () => {
+          return generateTypes()
+        },
+      },
```

Comment:
Can't this be simplified to just be task: generateTypes? I realize all the other type generation tasks look the same, but if it works, see it as an opportunity to fix them all ;)

```suggestion
        task: generateTypes
```